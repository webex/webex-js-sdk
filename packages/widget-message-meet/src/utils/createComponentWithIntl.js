import React from 'react';
import renderer from 'react-test-renderer';
import {IntlProvider} from 'react-intl';

function createComponentWithIntl(children, props = {locale: `en`}) {
  return renderer.create(
    <IntlProvider {...props}>
      {children}
    </IntlProvider>
  );
}

export default createComponentWithIntl;
