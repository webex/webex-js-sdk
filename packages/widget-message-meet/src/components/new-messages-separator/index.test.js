import React from 'react';
import createComponentWithIntl from '../../utils/createComponentWithIntl';

import NewMessagesSeparator from '.';

describe(`NewMessagesSeparator component`, () => {
  it(`renders properly`, () => {
    const component = createComponentWithIntl(
      <NewMessagesSeparator
        primaryText={`Testing 123`}
      />
    );
    expect(component).toMatchSnapshot();
  });
});
