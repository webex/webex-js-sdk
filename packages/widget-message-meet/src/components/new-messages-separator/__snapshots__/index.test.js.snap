exports[`NewMessagesSeparator component renders properly 1`] = `
<div>
  <div
    className="separator informative">
    <p
      className="separator-text informative">
      <span>
        NEW MESSAGES
      </span>
    </p>
  </div>
</div>
`;
